### PR TITLE
JMLL-399 Update leaver details page

### PR DESCRIPTION
**Pull request by jafacakes2011**

### CODE REVIEW

File: leavers/views/leaver.py [hunk 1]
```diff
@@ -1,41 +1,172 @@
...
+
+        people_finder_results = search_people_finder(search_term=email)
+        if len(people_finder_results) > 0:
+            person = people_finder_results[0]
+
+            job_title = ""
+            directorate = ""
+            if "roles" in person and len(person["roles"]) > 0:
+                job_title = person["roles"][0]["job_title"]
+                directorate = person["roles"][0]["team"]["name"]
```
Comment (jafacakes2011) [inline leavers/views/leaver.py:75]: @rossmiller is the Team name here the Directorate?

File: leavers/views/leaver.py [hunk 2]
```diff
@@ -1,41 +1,172 @@
...
+
+            # TODO: Map values to the blank leaver details
```
Comment (jafacakes2011) [inline leavers/views/leaver.py:77]: We need to get all the field values from their different locations

File: leavers/forms.py [hunk 3]
```diff
@@ -134,44 +124,42 @@ class LeaverConfirmationForm(GovFormattedForm):
...
-    # declaration = forms.BooleanField(
-    #     label="I confirm that all information is up to date and correct"
-    # )
 
 
-class PersonalDetailsForm(GovFormattedForm):
-    personal_email_address = forms.EmailField()
-    address = forms.CharField(
-        label="Find the leaver using their name or email",
+class LeaverUpdateForm(GovFormattedForm):
```
Comment (jafacakes2011) [inline leavers/forms.py:129]: **Task:** Update fields to be required/not required

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**